packages/components/src/dropdown-menu/index.native.js

```diff
@@ -124,7 +124,6 @@ function DropdownMenu( {
 						aria-haspopup="true"
 						aria-expanded={ isOpen }
 						label={ label }
-						showTooltip
```

Comment:
`showTooltip` defaulting to `true` was likely carried over from the web implementation where the prop controls whether _hovering should display the button's tooltip_. Given that hovering is an interaction that is not possible on touch device currently, its presence makes less sense. 

With the previous native implementation of Tooltip being a no-op, this prop resulted in no noticeable impact. However, in the native Tooltip implemented in this PR, this prop would now cause the tooltip to be displayed on initial render for every button in the dropdown menu.